### ISSUE TITLE
Volume2 causes USB DAC to lag randomly 

### ISSUE THREAD

**Issue by Akanaoi** [irzyxa/Volume2#155]
Hello, I am using a USB DAC CX-Pro CX31993. And most of the time while I use volume2 it lags the DAC. I stopped and use the 3RVX to use the CTRL + UP/DOWN to control the volume of my PC. 

So far it's been 4 days since I switched temporarily, the USB DAC never had any lag issues like controling the volume via shortcut or directly from windows volume slider. 

Windows 10 21H2
 V2 - 1.1.7.438

**Comment by irzyxa** [comment]
I have just ordered one on AliExpress, so I can test it when it will come.

**Comment by irzyxa** [comment]
It is here. The first test shows nothing, it works without any lags. How lags look like? Could you add some video?

**Comment by Akanaoi** [comment]
> 

I can't take a video but it may be comprehensible if I say specifically.

The lag happens randomly, and the only thing I notice is when using the volume, it reacts very slow. When I play a song, MP3, DSD, FLAC file via any audio player such as MusicBee, JetAudio, Groove Music, it doesn't wanna play during the lag issue. It also happened while a music was playing. Playing the next song or waiting for it to finish until a new song that supposed to be start playing is not playing anymore unless I replug the DAC.

**Comment by irzyxa** [comment]
Have you tried the latest version of Volume2?

**Comment by irzyxa** [comment]
And could you please send your Config.ini file?

**Comment by Akanaoi** [comment]
> Have you tried the latest version of Volume2?



> And could you please send your Config.ini file?

I will try again to use the latest Beta. I'll recheck again if the issue will re-occur.

Here's the config content.

`;Warning! Editing this file, can cause Volume² to stop work
[Languages]
LangFileName=English.lng
LangName=English (US)

[UserVariables]
ImmediateHideVolumeControl=0
ExKatMouseClasses=TRichViewEdit
ExKatMouseExcludeClasses=Windows.UI.Core.CoreWindow
ExKatMouseScrollCount=1
UseRemoteServer=0
RemoteServerPort=22222
RemoteServerUdpPort=22221
DelayBeforePlaySound=500
DelayBeforeRestartMouseHook=-1
UseAutoRestartHooks=0
DelayBeforeHideHint=2500
DelayBeforeHideEdgeWnd=5000
LowLevelHooksTimeout=10000
LowLevelShortKeys=Pause,"Win + 0","Win + 1","Win + 2","Win + 3","Win + 4","Win + 5","Win + 6","Win + 7","Win + 8","Win + 9","Win + Up","Win + Down","Win + Left","Win + Right",F12
ExScrollObjects=ActualTools_MultiMonitorTaskbar,DockBackgroundClassTrueStretch,DFTaskbar*
DefaultDeviceRole=-1
DetectActiveMonitorMethod=1
UseLockDevicesVolume=0
DevicesSort=1
AutoClosePeakMeter=0
WindowCaptionExcludeAppList=

[System]
SystemVolume=11
SystemMute=0
UseAlternativeMouseWheel=0
SoundName=No sound
UseAutoUpdate=0
UseBetaUpdate=0
NotifyUsb=0
NotifyCD=0
NotifyBright=0
NotifyVolume=1
NotifyVolumeType=0
SystemShowBaloonHint=1
AskOnExit=0
ShowStandartRegulator=0
UseInverse=0
RestoreSystemVolume=0
UseLongClick=0

[MouseMoveVolumeControl]
EdgeLeft=1
EdgeRight=0
EdgeTop=0
EdgeBottom=0
EdgeHintInfo=%D: %V%
EdgeShowToolTip=0
UseMoveWatch=0
TypeControlEdge=0
MoveStartStep=100
MoveStep=50
UseDisableSoundMiddleButton=0
EdgeZoneWidth=10
EdgeShowOSD=1
ShowEdgeWindow=1
ShowEdgeWindowWhenVolumeChanged=0
EdgeBackColorOn=-16711936
EdgeBackColorOff=-65536
EdgeDisableWhenFullScreen=0

[Main]
AltDeviceScrollKey=0
UseAltDevice=1
MethodControl=0
ChangeVolumeFileName=
AltChangeVolumeFileName=
OffVolumeControlOnFullScreen=0
DisableHotKeysInFullScreen=0
AutoRun=1
ScheduleAutoRun=0
DeviceId=~
ChangeVolumeOption=0
AltDeviceId={0.0.0.00000000}.{8e3343e1-564b-4e26-9def-285272ca036e}
AltChangeVolumeOption=0
UnMuteOnVolumeChange=0
UnMuteOnVolumeIncrease=0
VolSpeed=35

[MouseActions]
UseDisableVolume=0
DisableVolumeMouseKey=0
DisableVolumeKey=0
ScrollObject=1
UseMouseScroll=1
UseDisableSoundMiddleButton=0
LeftMouseEvent=1
DblLeftMouseEvent=4
MidleMouseEvent=3
DisableMouseScrollInFullScreen=1
ScrollKey=0
UseScrollKey=0
MuteOnZeroVolume=0

[OSD]
OsdHintInfo=%D: %V%
OSDStepWidth=30
OSDSpaceWidth=1
OSDBorderSize=2
OSDContourSize=2
OSDProgressFadeSteps=25
OSDProgressShowSteps=100
OSDBorderColor=-16777216
OSDTextCountorColor=-16777216
OSDFontColorOn=-16711936
OSDFontColorOff=-16776961
OSDBackColorOn=-16711936
OSDBackColorOff=-16776961
OSDBackColorOnTo=-16711936
TrackBarColorOnFrom=-5308791
TrackBarColorOnTo=-1
TrackBarColorOffFrom=-47546
TrackBarColorOffTo=-1
OSDBackColorOffTo=-16776961
OSDFontSize=32
OSDShowTime=500
OSDTextBorder=1
OSDTextDelineate=1
OSDTextBlocks=1
OSDTextVisible=1
OSDFontName=Verdana
OSDFontBold=0
OSDFontItalic=0
OSDStyle=1
OSDOrientation=0
OsdWhileDisplaing=0
CurOSDHeight=100
CurOSDWidth=500
OSDTrackBarHeight=100
OSDTrackBarWidth=27
UseLabelOSDTrackBar=0
UseOSDTrackBarMousePosition=1
OSDTransparent=0
OSDShowFullBlocks=0
UseOSD=1
OSDSkinName=Windows 10 v2
DisableOSDInFullScreen=0
MonitorIndex=0
OSDStretchPercent=100
OSDPosition=109

[InterfaceColors]
COLOR_BORDER=14277081
COLOR_SELECTED=15581836
COLOR_BACKGROUND=16777215

[Skins]
SkinName=Volume2 Default Light
AltSkinName=No icon

[AdvSettings]
UseBalloonHint=0
BalloonHintInfo=%D: %V%
StyleHint=1

[SystemTray]
ShowTrayHintOnVolumeChange=0

[TrayVolumeControl]
TimeAutoHideVolumeControl=5000

[ShortKeys]
Pause=3
Pause_Parameter=
Ctrl_Up=5
Ctrl_Up_Parameter=
Ctrl_Down=6
Ctrl_Down_Parameter=
`

**Comment by irzyxa** [comment]
Okay, what is your mouse? Perhaps this is your situation: https://github.com/irzyxa/Volume2/wiki/FAQ#7-incompatibility

**Comment by Akanaoi** [comment]
> Okay, what is your mouse? Perhaps this is your situation: https://github.com/irzyxa/Volume2/wiki/FAQ#7-incompatibility

I am using a Razer Deathadder V2 mini (No Synapse Software)

**Comment by irzyxa** [comment]
And this happens not always, correct?

**Comment by Akanaoi** [comment]
> And this happens not always, correct?

Yes. Very random. The lag is hard to reproduce because I can only confirm the lag happened if I set the the volume low/high.

**Comment by Akanaoi** [comment]
Hi, I would like to report the past 3 days since using the latest Beta.

It happened 1-3 times. It also messed up the volume's percentage when the lag happened. I was using the 35 steps volume changing speed. The LEFT volume became low so when I look up on the balance option of the volume in windows settings, it became R 15 / L 13.
